.github/workflows/fedora-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Check build for Fedora.
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  package:
+    container: fedora:latest
```

Comment:
I wonder if we can decrease the run time by using a image that already has @development-tools @rpm-development-tools installed? (If that even exists)

Reply:
I've checked the [official fedora docker.hub](https://hub.docker.com/_/fedora) images and there is no docker images with @development-tools installed. Also can't find anything similar at docker.hub.